packages/eslint-plugin-svelte/src/rules/no-navigation-without-resolve.ts

```diff
@@ -84,6 +84,26 @@ export default createRule('no-navigation-without-resolve', {
 					}
 				}
 			},
+			SvelteShorthandAttribute(node) {
+				if (
+					context.options[0]?.ignoreLinks === true ||
```

Comment:
Could you extract this outside of the `return {...}` and store it in a variable?
This will prevent us from having to modify the default option in multiple places in the future and forgetting to do so.

Reply:
Done :)